drivers/scheduler/k8s/k8s.go

```diff
@@ -670,7 +670,7 @@ func (k *k8s) WaitForRunning(ctx *scheduler.Context) error {
 
 			logrus.Infof("[%v] Validated Rule: %v", ctx.App.Key, svc.Name)
 		} else if obj, ok := spec.(*v1.Pod); ok {
-			if err := k8sOps.ValidatePod(obj); err != nil {
+			if err := k8sOps.ValidatePod(obj, timeout*2, retryInterval); err != nil {
```

Comment:
why `*2` here?

Reply:
Just use the given input timeout to WaitForRunning. 10 minutes was too much and was used to workaround an issue with the wordpress app.